### PR TITLE
feat: display profile name in navbar if available 

### PR DESCRIPTION
**Pull request by navinkarkera**
feat: workflow to publish package to opencraft npm

* `Private-ref`: [BB-8772](https://tasks.opencraft.com/browse/BB-8772)

### CODE REVIEW

File: example/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { initialize, getConfig, subscribe, APP_READY } from '@edx/frontend-platform';
 import { AppContext, AppProvider } from '@edx/frontend-platform/react';
-import Header from '@edx/frontend-component-header';
+import Header from '@opencraft/frontend-component-header';
```
Comment (CefBoud) [inline example/index.js:7]: Since this fork is installed using npm aliasing, does it make sense to replace `@edx` with `@opencraft`?

Comment (navinkarkera) [reply example/index.js:7]: No, I think it should refer itself with the correct name as npm alias will manage references in the repository which uses it. We did the same for [footer](https://github.com/open-craft/frontend-component-footer/commit/b49e7bf686331eccb2a7cdb138b72b219560eff1#diff-25eb80999c2e9f9cd03d36d2baf1364ec623fa5e4aed179df535cf3047dca8f2L7).

**Review by CefBoud — APPROVED** [review]

**Comment by Cup0fCoffee** [comment]
@navinkarkera Can you please remember to link the upstream PRs and ticket ID in PRs? It makes it much easier to find the ticket, instead of having to remember which epic it is from and looking through the list of tickets.

Same applies to the upstream PRs. E.g. this PR - https://github.com/openedx/frontend-component-header/pull/483 - doesn't have a link to the ticket, nor any mention of it in the branch or title.

**Comment by navinkarkera** [comment]
@Cup0fCoffee Yep, I did add all the details to main PR: https://github.com/openedx/edx-platform/pull/34493 but missed to include it in the sub PRs. Updated them.

**Pull request merged**